CONTRIBUTING.md

```diff
@@ -728,7 +715,7 @@ def sqrt(x: Union[float, int]) -> Union[float, int]:
     return math.sqrt(x)
 ```
 
-### TypeScript
+### TypeScript Typing
```

Comment:
This feels a bit redundant imo

Reply:
The `Typescript in Linting` and `Typescript in Typescript` are in conflict, so the Table of Contents isn't `typescript` in `Linting` section


<img width="618" alt="image" src="https://user-images.githubusercontent.com/2016594/146484690-c0375b13-2caa-41d0-9708-342ef8fbd06c.png">